src/Functions.js

```diff
@@ -0,0 +1,18 @@
+export const DateToString = (date) => {
```

Comment:
function names can be camelCase, only React Components are recommended to be Pascal Cased and a requirement of react

Reply:
I have fixed this.